src/board/Cell.test.jsx

```diff
@@ -0,0 +1,25 @@
+import { render } from "@testing-library/react";
+import Cell from "./Cell";
+import userEvent from "@testing-library/user-event";
+
+test("display a value in a cell", () => {
+  const { getByText } = render(
+    <Cell cellValue={"abc"} handleClick={jest.fn()} />
+  );
+
+  expect(getByText("abc")).toBeVisible();
+});
+
+test("handles click", () => {
+  const handleClickMock = jest.fn();
+  const cellValue = "abc";
+
+  const { getByText } = render(
+    <Cell cellValue={cellValue} handleClick={handleClickMock} />
+  );
+
+  const cell = getByText("abc");
+  userEvent.click(cell);
+
```

Comment:
I believe the best practice is using screen.getByText instead of pulling the functions from the render.